### ISSUE TITLE
Cannot install lts-gallium

### ISSUE THREAD

**Issue by sblask** [asdf-vm/asdf-nodejs#294]
I had my builds start to fail today with:

```
$ asdf install nodejs lts-gallium
Installing alias lts-gallium as 16.14.1
Trying to update node-build... ok
node-build: definition not found: 16.14.1
Could not install version 16.14.1
```

I run asdf from tag v0.9.0 and because I run in Docker I assume everything else is up to date.
When I check https://nodejs.org/en/download/releases/ there is no 16.14.1 and:

```
$ asdf list-all nodejs | grep 16.14
16.14.0
```

 I could not find where `lts-gallium` is defined or retrieved from.

**Comment by augustobmoura** [comment]
On the official releases [gallium is actually 16.14.1](https://nodejs.org/dist/latest-gallium/)

**Comment by sblask** [comment]
@augustobmoura so the `lts-gallium` alias is based on that? node-build doesn't seem to know about it yet: https://github.com/nodenv/node-build/tree/master/share/node-build

Any idea how to fix it?

**Comment by augustobmoura** [comment]
I think I see the problem, we are correctly identifying the latest lts as 16.14.1, but because the version is too recent nodebuild didn't even generated the definitions for this version, (the [action runs every 4 hours](https://github.com/nodenv/node-build/blob/9b7b5f0fa19b47b728eaf8a529f0151232618171/.github/workflows/scrape-definitions.yml#L3)), and even after scraping new versions a new PR still needs to be accepted.

I think the correct solution would be to install a version as lts only is it's definition is available on nodebuild, instead of trying to install the latest one every time. A solution while we wait for the definition to be merged on master is to just install 16.14.0 and link it as the lts version, something like:

``` bash
asdf install nodejs 16.14.0
ln -s "$ASDF_DIR/installs/16.14.0" "$ASDF_DIR/installs/lts-gallium"
```

I will try to work on a permanent solution for that problem, together with some other changes to the aliasing logic
